Docs/README.md

```diff
@@ -64,65 +72,101 @@ Splice reads:                                  4161455
 Reads mapped in proper pairs:                  21856264
 Proper-paired reads map to different chrom:    7648
 ```
+The MultiQC plot of this information can be a bit overwhelming at first but it displayes the above information simultaneously for all the samples in your report. With each sample represented as a dot. 
```

Comment:
_MultiQC plots each of these statistics in a dot plot (each sample is a dot - hover to see the sample highlighted across all fields)._